exe/etcd-env

```diff
@@ -5,14 +5,17 @@ require 'json'
 require 'net/http'
 
 def usage
-  puts "Usage: #{$PROGRAM_NAME} <etcd key> <command>"
+  puts "Usage: #{$PROGRAM_NAME} <etcd key> [ <etcd key> ... ] -- <command>"
 end
 
-def env_vars(key)
-  response = fetch "http://#{etcd_host}:#{etcd_port}/v2/keys#{key}"
-  (JSON.parse(response)['node']['nodes'] || [])
-    .map { |var| [var['key'].split('/').last, var['value']] }
-    .to_h
+def env_vars(keys)
+  keys.map! do |key|
```

Comment:
Something like this:

```ruby
keys.reduce({}) do |memo, key|
  response = fetch "http://#{etcd_host}:#{etcd_port}/v2/keys#{key}"
  vars = (JSON.parse(response)['node']['nodes'] || [])
      .map { |var| [var['key'].split('/').last, var['value']] }
      .to_h
  memo.merge vars
end
```

See http://ruby-doc.org/core-2.2.1/Enumerable.html#method-i-reduce